### ISSUE TITLE
Escape node id in graphviz dot diagram

### ISSUE THREAD

**Issue by Place1** [dominikbraun/graph#44]
Hello, this is a really cool library. Thanks so much for creating it!

I found a small bug in the [`dotTemplate`](https://git.getjames.me/james/dotfiles/blob/master/go/pkg/mod/github.com/dominikbraun/graph%40v0.10.0/draw/draw.go#L15-L16) used to render graphviz dot diagrams; it doesn't quote node ids or escape special characters.

Could we improve support by adding quotes to the dotTemplate and escaping any quotes in the node id?

Here's an id i've got in my graph for reference: `/home/folder:hello.world`.

If i manually quote the strings in the generated dot diagram it works.

```
strict digraph {

	"/home/folder:hello.world" -> "something/else" [  weight=0 ];

}
```

**Comment by dominikbraun** [comment]
Thanks for reporting this! Shouldn't be too hard to support nodes containing whitespaces and special characters.

**Comment by dominikbraun** [comment]
@Place1 The fix has been released in [v0.11.0](https://github.com/dominikbraun/graph/releases/tag/v0.11.0).

**Comment by Place1** [comment]
Thanks so much! Amazing
